pkg/sidecar/identity/member_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (C) 2021 ScyllaDB
+
+package identity
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func TestMember_GetSeeds(t *testing.T) {
+	createPodAndSvc := func(name, ip string, creationTimestamp time.Time) (*corev1.Pod, *corev1.Service) {
```

Comment:
nit: separate functions might be cleaner if the args are not connected

Reply:
they do share the name, it's a private function only in this test so i'll leave it as it is.